apps/studio/components/interfaces/SQLEditor/OngoingQueriesPanel.tsx

```diff
@@ -70,12 +72,12 @@ export const OngoingQueriesPanel = ({ visible, onClose }: OngoingQueriesPanel) =
 
   const closePanel = () => {
     setParams({ viewOngoingQueries: undefined })
-    onClose()
+    appState.setOnGoingQueriesPanelOpen(false)
   }
 
   return (
     <>
-      <Sheet open={visible} onOpenChange={() => closePanel()}>
+      <Sheet open={appState.ongoingQueriesPanelOpen} onOpenChange={() => closePanel()}>
```

Comment:
don't know why this component wasn't dealing with it's own state.
removed the prop for open/visible